rahulmalik87/pstress
# Add logging options for recent N queries

### This update introduces new query logging option:

**--log-N-queries=N :** Saves the last N queries to tmp/node.3306/step-1/thread-0/recent-<N>.sql. Defaults to 5 if not specified.

> Fallback: If no logging option is selected, the default is to log the last 5 queries.

**--log-all-queries :** Saves a full query log to tmp/node.3306/step-1/thread-0.sql.

**How to Test:**

1. Enable --log-all-queries and confirm the full log is saved.
2. Set --log-N-queries=10 and check if only the last 10 queries are logged.
3. Ensure default behavior logs the last 5 queries when no options are selected.
